microapps/OfflineMapAreasApp/app/src/main/java/com/arcgismaps/toolkit/offlinemapareasapp/screens/MainScreen.kt

```diff
@@ -18,39 +18,62 @@
 
 package com.arcgismaps.toolkit.offlinemapareasapp.screens
 
-import androidx.compose.foundation.layout.Column
+import androidx.compose.animation.AnimatedVisibility
+import androidx.compose.animation.slideInVertically
+import androidx.compose.animation.slideOutVertically
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.BottomSheetScaffold
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SheetValue
+import androidx.compose.material3.rememberBottomSheetScaffoldState
+import androidx.compose.material3.rememberStandardBottomSheetState
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
-import androidx.compose.runtime.mutableStateOf
-import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
-import com.arcgismaps.mapping.ArcGISMap
-import com.arcgismaps.mapping.BasemapStyle
-import com.arcgismaps.mapping.Viewpoint
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.LoadStatus
 import com.arcgismaps.toolkit.geoviewcompose.MapView
 import com.arcgismaps.toolkit.offline.OfflineMapAreas
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun MainScreen() {
-    val arcGISMap by remember {
-        mutableStateOf(
-            ArcGISMap(BasemapStyle.ArcGISTopographic).apply {
-                initialViewpoint = Viewpoint(
-                    latitude = 39.8,
-                    longitude = -98.6,
-                    scale = 10e7
-                )
-            }
+fun MainScreen(viewModel: OfflineViewModel = viewModel()) {
+    val loadState by viewModel.arcGISMap.loadStatus.collectAsState()
+
+    val scaffoldState = rememberBottomSheetScaffoldState(
+        bottomSheetState = rememberStandardBottomSheetState(
+            initialValue = SheetValue.Expanded,
+            skipHiddenState = true
         )
-    }
-    Column {
+    )
+
+    BottomSheetScaffold(
+        sheetContent = {
+            AnimatedVisibility(
+                visible = loadState is LoadStatus.Loaded,
+                enter = slideInVertically { h -> h },
+                exit = slideOutVertically { h -> h },
+                label = "trace tool",
+                modifier = Modifier.heightIn(min = 0.dp, max = 350.dp)
+            ) {
+                OfflineMapAreas(viewModel.offlineMapState)
+            }
```

Comment:
With the above changes. no need to constraint height, sheet height will respect the Toolkit Composable height. 
```suggestion
            OfflineMapAreas(viewModel.offlineMapState)
```